crates/conf/macros/tests/fails/bad_check_closure_parse.rs

```diff
@@ -0,0 +1,11 @@
+use conf_macros::Config;
+
+#[derive(Config)]
+pub struct TestConfig {
+    pub field1: u32,
+    #[check(foobar)]
+    pub field2: u32,
+    pub field3: u32,
+}
+
+fn main() {}
```

Comment:
Please add a new line here and to every text-base file which does not already end with a new line character.